docs/root/intro/arch_overview/upstream/aggregate_cluster.rst

```diff
@@ -0,0 +1,103 @@
+| 71%                   | 1%                    | 0%                    | 100%                  | 100%                  | 100%               | 0%                   |
++-----------------------+-----------------------+-----------------------+-----------------------+-----------------------+--------------------+----------------------+
+| 71%                   | 0%                    | 0%                    | 100%                  | 100%                  | 99%                | 1%                   |
++-----------------------+-----------------------+-----------------------+-----------------------+-----------------------+--------------------+----------------------+
+| 50%                   | 0%                    | 0%                    | 50%                   | 0%                    | 70%                | 30%                  |
++-----------------------+-----------------------+-----------------------+-----------------------+-----------------------+--------------------+----------------------+
+| 20%                   | 20%                   | 10%                   | 25%                   | 25%                   | 70%                | 30%                  |
++-----------------------+-----------------------+-----------------------+-----------------------+-----------------------+--------------------+----------------------+
+| 0%                    | 0%                    | 0%                    | 100%                  | 0%                    | 0%                 | 100%                 |
++-----------------------+-----------------------+-----------------------+-----------------------+-----------------------+--------------------+----------------------+
+| 0%                    | 0%                    | 0%                    | 72%                   | 0%                    | 0%                 | 100%                 |
++-----------------------+-----------------------+-----------------------+-----------------------+-----------------------+--------------------+----------------------+
+| 20%                   | 0%                    | 0%                    | 20%                   | 0%                    | 50%                | 50%                  |
++-----------------------+-----------------------+-----------------------+-----------------------+-----------------------+--------------------+----------------------+
+
+
+Note: The above load balancing uses default :ref:`overprovisioning factor <arch_overview_load_balancing_overprovisioning_factor>` which is 1.4.
+
+The example shows how the aggregate cluster level load balancer selects the cluster. E.g.,
+healths of {{20, 20, 10}, {25, 25}} would result in a priority load of {70%, 30%} of traffic. When normalized total health drops below 100, traffic is distributed after normalizing
+the levels' health scores to that sub-100 total. E.g. healths of {{20, 0, 0}, {20, 0}} (yielding a normalized
+total health of 56) would be normalized, and result in a priority load of {50, 50} of traffic.
```

Comment:
Where does 56 come from?

Reply:
20*1.4 + 20*1.4. Added pseudo algorithms to explain the calculation.